apps/server/src/shared/repo/filerecord/filerecord.repo.integration.spec.ts

```diff
@@ -78,7 +78,9 @@ describe('FileRecordRepo', () => {
 		});
 	});
 
-	describe('save', () => {
+	// This test must be skipped until the migration to filerecords is finished because the automatic update of timestamps is disabled for this job.
+	// TODO: Unskip after migration.
```

Comment:
Same mark with easy found key word. 
// BC-1777 - temporary method for file sync job